docs/brexit-checker-content-changes.md

```diff
@@ -18,7 +18,11 @@ Actions are defined in [an `actions.yaml` file](https://github.com/alphagov/find
 GOOGLE_SHEET_ID="1BxiMVs0XRA5nFMdKvBdBZjgmUUqptlbs74OgvE2upms"
 ```
 
-3. Enable the Google Drive API by generating a `credentials.json` file from the API and saving it in the root directory of the repo.  Instructions for this can be found [here](https://developers.google.com/drive/api/v3/quickstart/ruby).  You will not need to do this again when running the rake task in future as long as you have `credentials.json`.
+3. Enable the Google Drive API by generating a `credentials.json` file from the API and saving it in the root directory of the repo.  Instructions for this can be found [here](https://developers.google.com/drive/api/v3/quickstart/ruby#step_1_turn_on_the). You will be prompted for the following information:
```

Comment:
```suggestion
3. Enable the Google Drive API by generating a `credentials.json` file from the API and saving it in the root directory of the repo.  Instructions for this can be found [here](https://developers.google.com/drive/api/v3/quickstart/ruby#step_1_turn_on_the) (click the "Enable the Drive API" button). You will be prompted for the following information:
```